README.md

```diff
@@ -53,7 +53,9 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+weather = "sunny"; will evaluate to the string 'sunny' since only one '=' is being used which will assign the variable weather a value of "sunny".
+
+weather === "sunny"; will return undefined. Since we are using strict equality both sides have to be identical in data type and value.
```

Comment:
This is incorrect, though. We just assigned `weather` the value `"sunny"` in the previous line, so the two are equal; this expression should evaluate to `true`.